local-running/core.local.secrets.template.yaml

```diff
@@ -7,22 +7,22 @@ core:
     # This test encryption key is only used locally
     jarEncryptionKey: '{"kty": "RSA","n": "0465qJwo8nCkC2tvV4niuWF6IM6pNjmeYszhTwHPY609-HVAtO8PoRLUyA86rzQ-QzbT7XxbzCjfyRXoRFOGleZqTuwlc25ezDxV58bhecPiWFMaFYOS1W7zIDsVFo37gjjvtkcD6OqK8PKAv6n5tUphjDCcnnmpTMIyGAnzmQCbSkJWu6V_gc3tirAugXoZukMCohxw3_-c6prhMN0smDNv0qWmva3oqokabePwe1OS72DXyXR-TPd_Dtz4-tRr9jvZwHulX4Zcs1BBbjBpIim3WNY8asv9yjlBxkdt-nckhCMZekPuT7xWSTrvccB_fnnSUgEQW_5irLNdnr5MWQ","e": "AQAB","d": "R7db77EUXm9Z5PckFPhMsAw_0uxLx3KL4zFR9a8G52LcnaPd7hgWY8OW90TWX3tl2fI1sI2WLuTyS-ANz0_DUYB9LEj4PyO8ph6T_pDYcac6TTfKoV1rgsfsSlJNCUrTZmc3t8JmLGiJ9jk29QJBgLf4zoXSKhTTFA7G3YhOGGDS2MS3ZrHMJBnjZYYHAyrCxJuGwZBiSiMwhie7OPpZYHpofCwK9PlupduNKKHQfHlzM-d07lit6u98hAZE555mIFxsGssDlOZ8JsHK6LknYqAjFkHvJGuZ28s_AyTanJ_aGngphuNm9f71eIB1LKgkBdYCBjrAxYkkTkKmSPhYkw","p": "8oxAsAsoYjwxkCLgCgolXOb4SOQevXntNGvcl_h_FUZOofw71PEcmrzRSukkUD8GDS8Cnb-GejtzKWpHHvkNUHnoWd_Fpo-1yBNXZrytXYh65Fn9H7j1j7-cVGolNvjWhccVc8QaZbMRdUOXBsncaj6IF4_TG_fWO7mVyZLoupM","q": "30p3BamouZONQ5hXyN4qIdZTJoGjSxNGHCDf0EFjoqPsJ_gij1z8eOxWtk9YfP0-fpX5Ji5klYziB0H_UP0XFeBxYn3iAMWMCpamkR0CSVNdHj9ZADqW4huVynssHDLL3PbuG-btohuTnVBDikr9OVt185BitO_LDGPugad5NOM","dp": "zWMGxAwuvkDKl4twvA-rY-gSB2dkwH4m52UO_4Fl29YVcAXvhHpIBqAteonCPlGgBjnwtTQh5A_DjSZ54zhyldslbjWXYVwAStRWBqFvDO0tY4DwX5R4HKnc736S1OIShmX3PHirAcOI3QvHL_BvvWgGRi18TwqnVxvoQzEr8hE","dq": "rJAH28B6PLDjg1HAxeOr--p-6EtXvzVuAV8GmRNvxMmaQGlrjA_4rhs4CDzZzf6IxZtxqShBY7I1HBbNEfIGXUnPVqaieqYwzB4RLCSXF8PtOusKBTtCUAuxf4IhXMmE84eCDN4y2A0nz3OwPrglaqZcSZdUMTEAYtr4AAtFiLE","qi": "27L2sofeonh_IZ3G2YEEti-udMK55hc-38asC8_Y1WcK2LupegpwaLNYFa7ZmQQ9lj-0RVv8vn4LkMkgf8_FGbmK4mv9-ul9aTS7y8tkfGDTcvGjUb4LMyvNnd3iZzKuDB-A5ZbvGzXQOq70kvlPetPHfyzuZyU9ZyU_rr5P3H0","alg": "RS256"}' # pragma: allowlist secret
     # Test CI config
-    ci-config: '[
+    ciConfig: '[
       {"ci":"non-breaching","detectedScore":2,"checkedScore":-2,"returnCode":"non-breaching"},
       {"ci":"breaching","detectedScore":20,"checkedScore":-20,"returnCode":"breaching"},
       {"ci":"breaching-p1-only","detectedScore":7,"checkedScore":-7,"returnCode":"breaching-p1-only"},
       {"ci":"needs-enhanced-verification","detectedScore":20,"checkedScore":-20,"returnCode":"needs-enhanced-verification"},
       {"ci":"needs-alternate-doc","detectedScore":20,"checkedScore":-20,"returnCode":"needs-alternate-doc"},
       {"ci":"always-required","detectedScore":1,"checkedScore":-1,"returnCode":"always-required"}
     ]'
-    gov-uk-notify:
-      api-key: "NOTIFY_API_KEY"
+  govUkNotify:
+    apiKey: "NOTIFY_API_KEY" # pragma: allowlist secret
```

Comment:
Funny that `apiKey` is bad but `api-key` is fine 🙃

Reply:
so weird